git-hooks/pre-push

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Not used for now
+# remote="$1"
+# url="$2"
+
+# Create the logs directory if not exists. It is ignored by git anyway
+test -d logs || mkdir logs
+
+# Check if ansible lint is installed
+lint_version=$(ansible-lint --version 2>&1)
+echo "Check if ansible-lint is installed and accessible: $lint_version"
+
+# Run ansible lint on the installation playbooks
+ansible-lint install/playbooks/main.yml | tee logs/ansible-lint-install.log
+
+# Run ansible lint on the test playbooks
+ansible-lint tests/playbooks/main.yml | tee logs/ansible-lint-tests.log
```

Comment:
These tee commands overwrite the previous contents of those files.  I'm not sure if that is what you want. 
I'm not sure you even need to capture the output.  I guess you first have to answer the following:
"What should happen if the user doesn't have ansible-lint installed?"

I tend to not write git-hooks myself and instead use pip install pre-commit to handle things like linting for me.

Reply:
Yes, I want to overwrite the log files every time.
I had a look to pre-commit, and I decided to minimise the dependencies and write a shell script. If ansible-lint is not installed, you should not in theory, be able to push the code.
Since this script will be called by Jenkins, it will be easy to see if anyone push the code without running the hook first.